freqtrade/strategy/interface.py

```diff
@@ -246,7 +248,10 @@ def stop_loss_reached(self, current_rate: float, trade: Trade, current_time: dat
 
         trailing_stop = self.config.get('trailing_stop', False)
 
-        trade.adjust_stop_loss(trade.open_rate, self.stoploss, initial=True)
+        if force_stoploss == 0:
+            trade.adjust_stop_loss(trade.open_rate, self.stoploss, initial=True)
+        else:
+            trade.adjust_stop_loss(trade.open_rate, force_stoploss, initial=True)
```

Comment:
we could use 
```python
trade.adjust_stop_loss(trade.open_rate, force_stoploss if force_stoploss 
                                    else self.stoploss, initial=True)
```

i'm not sure if that's easier / mor readable though - so i'll leave that up to you